technical-reports/README.md

```diff
@@ -0,0 +1,58 @@
+# DTCG Technical Reports
```

Comment:
How would someone stumble upon this? Should we mention this directory in the root README and the contributing doc?